db/migrate/20151114125209_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,19 @@
+class CreateFriendlyIdSlugs < ActiveRecord::Migration
+  def change
+    create_table :friendly_id_slugs do |t|
```

Comment:
I am not entirely sure why the generator creates this table, I would think the `slug` columns added to the tables would be sufficient?